packages/docs/src/pages/en/getting-started/upgrade-guide.md

```diff
@@ -55,30 +55,49 @@ app.use(vuetify)
+- `stateless`, `clipped`, `clipped-right` and `app` props have been removed from v-navigation-drawer, v-app-bar and v-system-bar. The position in the markup determines the appearance. Use the `order="number"` prop to influence it manually.
+
 ### Theme
 
 - Multiple themes are now supported, so `light` / `dark` props have been removed from components. Use `v-theme-provider` to set the theme for a specific component tree.
+  - Components that previously had a `dark` prop, such as v-app-bar, now accept `theme="dark"` prop
 - Theme colors set their foreground text color automatically, if you were using `light` / `dark` to get a different text color you probably don't need it anymore.
 - The variant naming scheme has changed slightly, it is now a single word instead of two. For example, `primary darken-1` is now `primary-darken-1`.
+  - To use variant namings as value for `color` props, the variant you intend to use needs to be enabled in the theme under `theme.variations.colors`. e.g: `colors: ['primary']`
 - Color classes have been renamed:
   - Backgrounds have a `bg-` prefix, for example `.primary` is now `.bg-primary`.
   - Text colors have a `text-` prefix, for example `.primary--text` is now `.text-primary`.
   - Variants are no longer a separate class, for example `.primary--text.text--darken-1` is now `.text-primary-darken-1`.
 - The theme system now uses CSS variables internally, so `customProperties` is no longer required.
   - If you were using `customProperties` in v2, the naming scheme has changed from `--v-primary-base` to `--v-theme-primary`.
   - Custom properties are now also an rgb list instead of hex so `rgb()` or `rgba()` must be used to access them, for example `color: rgb(var(--v-theme-primary))` instead of `color: var(--v-primary-base)`.
+- Theme colors in the theme config are now nested inside a `color` property, e.g. `const myTheme = { theme: { themes: { light: { colors: { primary: '#ccc' } } } } }`
+- `$container-padding-x` is now 16px instead of 12px as in v2. You can replace it with `$spacer * 3` to get to the previous look
+
+### SCSS
+
+- `$headings` was merged with `$typography`: Access font-size of subtitle-2 with `map-get($typography, 'subtitle-2', 'size')`
+- If you imported variables from `~vuetify/src/styles/settings/_variables` in v2, you have to replace it with `~vuetify/_settings`
+- Component variables that previously lived in e.g. `~/vuetify/src/components/VIcon/VIcon.sass` can now be imported from `~vuetify/_settings` directly too
+- `$displayBreakpoints` no longer includes `{breakpoint}-only` variables (e.g. xs-only), use `@media #{map-get(v.$display-breakpoints, 'xs')}` instead
+- Replace `map-get(v.$transition, 'fast-in-fast-out)` with `$accelerated-easing`
+- `$vuetify.breakpoint` has been renamed to `$vuetify.display` and extended with [new properties](/features/display-and-platform/)
```

Comment:
Not related to sass

Reply:
Moved to Theme section

Reply:
Not really related to theme either. I guess layout is the closest section we already have for this

Reply:
moved